dts/bindings/sensor/ti,tmag5273.yaml

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) 2023, deveritec GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Texas Instruments TMAG5273 Low-Power Linear 3D Hall-Effect Sensor with an I2C interface.
+  See the specification for the default I2C address.
+  The specification of the sensor can be found at:
+    https://www.ti.com/lit/ds/symlink/tmag5273.pdf
+  The <zephyr/dt-bindings/sensor/tmag5273.h> file should be included in the
+  DeviceTree and it provides macros that can be used for configuration.
+
+compatible: "ti,tmag5273"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  operation-mode:
+    type: int
+    default: 0
+    enum:
+      - 0
+      - 1
```

Comment:
It was more a semantic choice: if it's not the standby-mode, which other mode is it?
The sensor also supports a "wakeup-and-sleep" mode, so using the enum here is intended to increase readability.

Reply:
As I was looking into the TMAG5170 (and since I now use the driver in another project were I need "wakeup-and-sleep" (or "duty-cycle"-mode, as it is called in the TMAG5170), there will be a third option in near future. But this is part of another PR.